_includes/navbar.html

```diff
@@ -56,6 +56,7 @@
                     <a href="#" class="dropdown-toggle" data-toggle="dropdown">About Apache<strong class="caret"></strong></a>
                     <ul class="dropdown-menu">
                         <li><a href="http://www.apache.org">The Apache Software Foundation</a></li>
+                        <li><a href="http://www.apache.org/licenses/">License</a></li>
```

Comment:
Link to http://www.apache.org/licenses/#2.0, since that's actually the relevant license we are using?